### PR TITLE
fix xcode workspace

### PR DESCRIPTION
**Pull request by scottdallamura**

### CODE REVIEW

File: Tasks/Xcode/task.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "version": {
         "Major": 2,
         "Minor": 0,
-        "Patch": 2
+        "Patch": 3
```
Comment (ericsciple) [inline Tasks/Xcode/task.json]: Let's change this to 4 instead. Trying out a patch for onedrive with patch version 3.

Comment (scottdallamura) [inline Tasks/Xcode/task.json]: Understood. Go for it :)

-----Original Message-----

**Comment by bryanmacfarlane** [comment]
bump it to 10 - I want some head room in the event we have a couple private patches for onedrive and others.  There was a lot of changes to this task so it wouldn't surprise me if there were a couple of issues.

**Comment by bryanmacfarlane** [comment]
Nevermind - just realized the 3 is for the rollback which is safe (what they had before).  4 is fine

**Pull request merged**